test/common/gristUrls.ts

```diff
@@ -68,29 +69,36 @@ describe('gristUrls', function() {
   describe('getHostType', function() {
     const defaultOptions = {
       baseDomain: 'getgrist.com',
-      pluginUrl: 'https://plugin.getgrist.com/path',
+      pluginUrl: 'https://plugin.getgrist.com',
     };
+
+    const oldEnv = new testUtils.EnvironmentSnapshot();
```

Comment:
This must be inside a `before` event.

Reply:
Done!

I assume that in DocApi2.ts, this should the same.